### DISCUSSION TITLE
关于建立多个docs的问题

### DISCUSSION THREAD

**Discussion by ghost** [wangchucheng/hugo-eureka#119] [Q&A]
问题：我想在导航栏，建立多个docs文档属性的栏目，但是建立多个栏目后，只有一个栏目有效果，如图所示：
![image](https://user-images.githubusercontent.com/81373184/121799964-cab9cb80-cc61-11eb-975b-40cd6e2ff37b.png)
像这张图片里一样，这是一个docs，它左边是章节目录，右边是页面目录，
但是在第二个docs里，就显示不出来章节目录和页面目录，请问怎么解决，或者说我的配置有问题？
![image](https://user-images.githubusercontent.com/81373184/121800018-11a7c100-cc62-11eb-9a7c-0b479a955a63.png)

**Comment by wangchucheng** [comment]
看起来是配置问题，你可以提供仓库的地址，大家有时间的话可能可以进行查看

**Reply by ghost** [reply]
https://github.com/kevin-zkx/hugoblog.git
麻烦您啦
